src/app/components/Project/Project.tsx

```diff
@@ -0,0 +1,86 @@
+import {
+  Box,
+  Card,
+  CardContent,
+  Typography,
+} from '@mui/material';
+import dynamic from 'next/dynamic';
+import { } from 'react';
+
+/**
+ * @todo Is there a better way to import the Link component from Material UI without getting the NextJS error:
+ *
+ * "Cannot access Link.prototype on the server. You cannot dot into a client module from a server component. You can only pass the imported name through."
+ */
+const ClientOnlyLink = dynamic(() => import('@mui/material/Link'), {
+  ssr: false,
+});
+
+interface Link {
+  title: string;
+  url: string;
+}
+
+interface ProjectProps {
+  title: string;
+  description: string;
+  links: Array<Link>;
+  partner?: string;
+  languages: Array<string>;
+  technologies?: Array<string>;
+  programAreas?: string;
+}
+
+export default function Project({
+  title,
+  description,
+  links,
+  partner,
+  languages,
+  technologies,
+  programAreas,
+}: ProjectProps) {
+  return (
+    <Box>
+      <Card sx={{ minWidth: 275 }}>
+        <CardContent>
+          <Typography variant="h4" component="div">
+            {title}
+          </Typography>
+          <Typography variant="body2">
+            {description}
+          </Typography>
+          <Typography variant="h6">
+            {
+              links.map(link => (
+                <ClientOnlyLink href={link.url} target="_blank" rel="noopener">{link.title}</ClientOnlyLink>
```

Comment:
I couldn't use the `Link` component directly here. If I did:

![image](https://github.com/OpenBCca/openbc-web/assets/16601729/7fe96778-2faa-4b04-b7a4-9b4c215a263d)

![image](https://github.com/OpenBCca/openbc-web/assets/16601729/681f3c4c-3b65-473b-ba5b-d80d18ccbc47)

The workaround here seemed to work, but admittedly there might be a better way that I'm not familiar with, given my lack of experience with `NextJS`

https://github.com/OpenBCca/openbc-web/blob/149e36664d1a9c2e27afe9557b764e3b2f3fa75d/src/app/components/Project/Project.tsx#L10-L17

Reply:
Addressed in [fe2900e](https://github.com/OpenBCca/openbc-web/pull/5/commits/fe2900ee8cf00db241c6870417aba992ca588c03)